charting/tests/mirror_touchIndicator.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+<link rel="stylesheet" href="../../../dijit/themes/claro/claro.css">
+
+<script>
+	dojoConfig = {
+		parseOnLoad : true,
+		has : {
+			'dojo-bidi' : true
+		}
+	}
+</script>
+<script src='../../../dojo/dojo.js'></script>
+
+<script>
+	require([ "dojox/charting/Chart", "dojox/charting/axis2d/Default",
+			"dojox/charting/plot2d/StackedAreas",
+			"dojox/charting/action2d/TouchZoomAndPan",
+			"dojox/charting/action2d/TouchIndicator",
+			"dojox/charting/themes/Wetland", "dojo/ready" ], function(Chart,
+			Default, StackedAreas, TouchZoomAndPan, TouchIndicator, Wetland,
+			ready) {
+		ready(function() {
```

Comment:
ready isn't something that should be needed post Dojo 1.8. You should just be able to use dojo/domReady! and then work with the code directly inside this block.